map_frame_name: "/map"
initialpose_topic_name: "/initialpose"
pub_arrive_flag_topic_name: "arrive_flag"
pub_location_marker_topic_name: "location_marker"
sub_initial_ctrl_position_topic_name: "initial_ctrl_pose"
sub_initial_ctrl_topic_name: "initial_ctrl"
sub_move_ctrl_position_topic_name: "move_ctrl_pose"
sub_move_ctrl_topic_name: "move_ctrl"
sub_location_stock_topic_name: "location_stock"
server_wake_wait_time: 100.0 # [sec] #
server_action_wait_time: 300.0 # [sec] #
move_service_name: "move_ctrl"
initial_service_name: "initial_ctrl"
stock_service_name: "stock_ctrl"
move_actionlib_name: "move_ctrl_actionlib"
